openapi/components/schemas/UserExists.yaml

```diff
@@ -4,12 +4,7 @@ properties:
     default: false
     description: Status if a user exist with that username or userId.
     type: boolean
-  nameOk:
-    default: false
-    description: Is the username valid?
-    type: boolean
 required:
   - userExists
-  - nameOk
```

Comment:
This is still included if you search by username.

Reply:
Re-added, but removed required flag